network/manager.go

```diff
@@ -382,6 +476,31 @@ func (nm *networkManager) GetEndpointInfo(networkId string, endpointId string) (
 	nm.Lock()
 	defer nm.Unlock()
 
+	if nm.IsStatelessCNIMode() {
+		cnilogger.Logger.Info("calling cns getEndpoint API")
+		endpointResponse, err := nm.CnsClient.GetEndpoint(context.TODO(), endpointId)
+		if err != nil {
+			cnilogger.Logger.Error("Get endpoint API returend with error", zap.Error(err))
+			return nil, errors.Wrapf(err, "Get endpoint API returend with error")
```

Comment:
@tamilmani1989 
I can check in here if the state is not found we return err=nil. But I'm afarid we might create other issues and prevent a cleanup.

Reply:
should check if error is `endpoint not found` and then ignore.. cannot ignore all other errors

Reply:
We should still return err here back to the cni/network.go Delete func so it can just exit gracefully the same as legacy CNI.